pkg/template/servicebroker/provision.go

```diff
@@ -240,19 +286,16 @@ func (b *Broker) Provision(instanceID string, preq *api.ProvisionRequest) *api.R
 		return api.BadRequest(kerrors.NewNotFound(templateapi.Resource("templates"), preq.ServiceID))
 	}
 
-	// TODO: enable SAR for template - at the moment I think this doesn't work
-	// properly because group information isn't populated in u.
-	/*
-		if err := util.Authorize(b.kc.Authorization().SubjectAccessReviews(), u, &authorization.ResourceAttributes{
-			Namespace: template.Namespace,
-			Verb:      "get",
-			Group:     templateapi.GroupName,
-			Resource:  "templates",
-			Name:      template.Name,
-		}); err != nil {
-			return api.Forbidden(err)
-		}
-	*/
+	// with groups in the user.Info vs. the username only form of auth, we can SAR for get access on template resources
+	if err := util.Authorize(b.kc.Authorization().SubjectAccessReviews(), u, &authorization.ResourceAttributes{
```

Comment:
on further reflection (and the lack of evidence that SC passing user info is imminent) i'm inclined to say we should comment this out for now w/ another "TODO - uncomment when SC is passing user info headers" (same PR as the others i guess).  But since this isn't going to merge for another week (when the next sprint starts) I guess we can wait and see.